python-backend/tests/tailings/resources/test_mine_tailings_resource.py

```diff
@@ -0,0 +1,45 @@
+import json, uuid, requests
+from tests.constants import TEST_MINE_GUID, TEST_MINE_GUID2, TEST_TAILINGS_STORAGE_FACILITY_NAME2, TEST_TAILINGS_STORAGE_FACILITY_NAME1
+
+
+# GET
+def test_get_mine_tailings_storage_facility_by_mine_guid(test_client, auth_headers):
+    get_resp = test_client.get('/mines/tailings?mine_guid=' + TEST_MINE_GUID, headers=auth_headers['full_auth_header'])
+    get_data = json.loads(get_resp.data.decode())
+    assert get_resp.status_code == 200
+    assert len(get_data['mine_storage_tailings_facilities']) == 1
+
+def test_post_mine_tailings_storage_facility_by_mine_guid(test_client, auth_headers):
+    get_resp = test_client.get('/mines/tailings?mine_guid=' + TEST_MINE_GUID, headers=auth_headers['full_auth_header'])
+    get_data = json.loads(get_resp.data.decode())
+    assert get_resp.status_code == 200
+    org_mine_tsf_list_len = len(get_data['mine_storage_tailings_facilities'])
+
+    post_resp = test_client.post('/mines/tailings', data={'mine_guid':TEST_MINE_GUID, 'tsf_name':TEST_TAILINGS_STORAGE_FACILITY_NAME2}, headers=auth_headers['full_auth_header'])
+    post_data = json.loads(post_resp.data.decode())
+    assert post_resp.status_code == 200
+    assert len(post_data['mine_tailings_storage_facilities']) == (org_mine_tsf_list_len +  1)
+    assert all(tsf['mine_guid'] == TEST_MINE_GUID for tsf in post_data['mine_tailings_storage_facilities'])
+
+### #appears that these tests effect each other, insert from above persists when this test runs org_exp_document_list_len is 2, only 1 inserted on setup
```

Comment:
Is there a proposed solution here?

Reply:
Is this blocking other people from getting stuff done? If there are dependencies for frontend and such, we can merge this in as long as we address these comments in the next PR or so before the end of sprint.

Reply:
no, this is a testing issue only. running the application normally doesn't cause any issues. this is mostly a werkzurg issue in how it handles/create api requests..